layouts/partials/cards/job_card/job_card.html

```diff
@@ -0,0 +1,56 @@
+
+{{ $path := printf "%s%s%s%s%s" "/content/jobs" "/" (lower .role) "/" (urlize (lower .jobData.name)) }}
+{{ $indexPath := printf "%s%s" $path "/_index.md" }}
+{{ $advancedGuide := printf "%s%s" $path "/advanced-guide.md" }}
+{{ $basicGuide := printf "%s%s" $path "/basic-guide.md" }}
+{{ $bis := printf "%s%s" $path "/best-in-slot.md" }}
+{{ $faq := printf "%s%s" $path "/faq.md" }}
+{{ $jobChanges := printf "%s%s" $path "/job-changes.md" }}
+{{ $levelingGuide := printf "%s%s" $path "/leveling-guide.md" }}
+{{ $openers := printf "%s%s" $path "/openers.md" }}
+{{ $skillsOverview := printf "%s%s" $path "/skills-overview.md" }}
+{{ $statPriority := printf "%s%s" $path "/stat-priority.md" }}
+
+{{ $pages := slice
+  (.siteVar.GetPage $levelingGuide)
+  (.siteVar.GetPage $basicGuide)
+  (.siteVar.GetPage $advancedGuide)
+  (.siteVar.GetPage $skillsOverview)
+  (.siteVar.GetPage $openers)
+  (.siteVar.GetPage $faq)
+  (.siteVar.GetPage $bis)
+  (.siteVar.GetPage $jobChanges)
+  (.siteVar.GetPage $statPriority)
+}}
+
+<div class="mt-0 lg:mt-12">
+  <div class="flex flex-col w-full py-3 justify-center items-start">
+    <div class="card card- space-y-3 w-full">
+      {{ if isset .jobData "name" }}
+        <div class="border-b border-card-dark pb-4">
+          {{ partial "cards/job_card/_header.html" .jobData }}
+        </div>
+      {{ end }}
+      <div>
+        {{ with .siteVar.GetPage $indexPath}}
+          {{ .Content }}
+        {{end}}
+      </div>
+      <div>
+        <h4 class="uppercase tracking-widest border-t border-black my-4 pt-4">Guides</h4>
+        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
+          {{ range $pages }}
+            {{ if .Path }}
+              <div>
+                <a href="{{ .Path }}" class="text-link-orange">{{ .Name }}</a>
```

Comment:
`.Permalink` is probably better to use here and should always exist as well.

Reply:
Resolved